torchao/quantization/GPTQ.py

```diff
@@ -1,1193 +0,0 @@
-# Copyright (c) Meta Platforms, Inc. and affiliates.
-# All rights reserved.
-#
-# This source code is licensed under the BSD-style license found in the
-# LICENSE file in the root directory of this source tree.
-
-
-# This source code is licensed under the license found in the
-# LICENSE file in the root directory of this source tree.
-
```

Comment:
Can we keep this file and keep the legacy imports for now? It will help us migrate some known use cases. E.g. something like
```
from torchao.quantization.quantized_linear import (
    Int4WeightOnlyQuantizer,
    Int8DynActInt4WeightQuantizer,
    ...
)
```